packages/website/hooks/use-payment.js

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react';
+
+import { userBillingSettings } from '../lib/api';
+import { plansAll } from '../components/contexts/plansContext';
+
+export const usePayment = () => {
```

Comment:
will you add a comment explaining what this is intended to be used for? How would another dev look at this and decide whether or not it's appropriate for their use case in another part of the codebase?